app/src/main/java/openfoodfacts/github/scrachx/openfood/network/OpenFoodAPIService.java

```diff
@@ -30,10 +30,10 @@
     String PRODUCT_API_COMMENT = "new android app";
 
 
-    @GET("api/v0/produit/{barcode}.json")
+    @GET("api/v0/product/{barcode}.json?fields=image_small_url,imageFrontUrl,image_ingredients_url,image_nutrition_url,url,code,traces_tags,ingredients_that_may_be_from_palm_oil_tags,additives_tags,allergens_hierarchy,manufacturing_places,nutriments,ingredients_from_palm_oil_tags,brands_tags,traces,categories,ingredients_text,product_name,generic_name,ingredients_from_or_that_may_be_from_palm_oil_n,serving_size,allergens,origins,stores,nutrition_grade_fr,nutrient_levels,countries,brands,packaging,labels,cities_tags,quantity,ingredients_from_palm_oil_n,image_url")
```

Comment:
I was considering going even more radically minimal for quickmode, only displaying what's actually shown

```image_small_url,code,product_name,nutrition_grade_fr,brands,quantity```